Cargo.toml

```diff
@@ -33,6 +33,9 @@ panic_handler = ["esp-idf-sys/panic_handler"]
 binstart = ["esp-idf-sys/binstart"]
 libstart = ["esp-idf-sys/libstart"]
 
+[patch.crates-io]
+esp-idf-sys = { path="../esp-idf-sys" }
```

Comment:
you can now directly reference your branch of this on github with git = and branch= , this will give the ci the chance to run correctly

Reply:
Done in 6f9a02fc6fdf545ebec16e27d07532b97f8753dd